providers/ietf/ietf.go

```diff
@@ -0,0 +1,23 @@
+package ietf
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("ietf", &IETF{})
```

Comment:
Understood. I used the Steam provider as a template which also does not follow the convention. I will submit a separate PR to correct it.